packages/frontend2/src/app/(new)/(other)/_components/base-scaling-filters.tsx

```diff
@@ -11,6 +11,7 @@ interface Props {
   items: CommonScalingEntry[]
   showRollupsOnly?: boolean
   additionalFiltersLeft?: React.ReactNode
+  // NOTE (torztomasz): I don't want to remove this as it will break other PR that @imxeno made. But I think it is not needed here and the behavior is wrong.
```

Comment:
Just create a spike task for @imxeno to decide whether we should remove it or not